pennylane/ops/qubit.py

```diff
@@ -1907,6 +1907,11 @@ def __init__(self, *params, control_wires=None, wires=None, control_values=None,
         target_dim = 2 ** len(wires)
         if len(U) != target_dim:
             raise ValueError(f"Input unitary must be of shape {(target_dim, target_dim)}")
+
+        # Saving for the circuit drawer
+        self.control_wires = control_wires
+        self.U = U
```

Comment:
If these attributes are required by the circuit drawer, we should probably document it somewhere.

Are there any other operations that are missing these attributes?

Reply:
> Are there any other operations that are missing these attributes?

They seem to be specific to `ControlledQubitUnitary` (at least with the current structure of PennyLane). Storing these two are required because:
* `control_wires` is merged into `self.wires` with the target wires at a later stage;
* the unitary is expanded to the entire subsystem using identities on the control qubits and only the expanded unitary is stored in `self.data`.

Therefore, introducing `self.control_wires` and `self.U` provides a convenient way to gather both the control wires and the unitary.

We could actually consider separating control and target wires natively for other more defined operations, as it can be convenient to have the separation be available easily.

>If these attributes are required by the circuit drawer, we should probably document it somewhere.

Indeed, at the moment they are well used by the circuit drawer, but it can easily be that other features will also use them. Any thoughts on where they could be documented? For attributes of operations, we don't seem to have any specific docstrings/docs.

Reply:
> Any thoughts on where they could be documented? 

This is a good question which I don't think has a good answer 🤔